hbase-client/src/main/java/org/apache/hadoop/hbase/client/RegionInfo.java

```diff
@@ -68,7 +68,7 @@
  *
  */
 @InterfaceAudience.Public
-public interface RegionInfo {
+public interface RegionInfo extends Comparable<RegionInfo> {
```

Comment:
Does this impact compatibility? If not, changes look good.

Reply:
No, it shouldn't. This is the same change that went into HBase 2.3.0+ in HBASE-23753.